npm/webpack-dev-server/cypress/e2e/angular.cy.ts

```diff
@@ -22,8 +22,8 @@ for (const project of WEBPACK_REACT) {
     it('should mount a passing test', () => {
       cy.visitApp()
       cy.contains('app.component.cy.ts').click()
-      cy.waitForSpecToFinish()
-      cy.get('.passed > .num').should('contain', 1)
+      cy.waitForSpecToFinish({ passCount: 1 })
```

Comment:
I am getting typescript errors from this call due to the version of the command in the `webpack-dev-server` not having any arguments for `waitForSpectToFinish`.  It is defined in `npm/webpack-dev-server/cypress/support/commands.ts`. The command that does have it is in the `app` package defined here: `packages/app/cypress/e2e/support/execute-spec.ts`

Reply:
Huh - for some reason I don't get these TS errors 🤔 maybe something is messed up with my IDE configuration. Good catch